static/app/views/replays/replays.tsx

```diff
@@ -60,48 +60,51 @@ class Replays extends React.Component<Props> {
 
   renderTable(replayList: Array<Replay>) {
     const {organization} = this.props;
-    return replayList?.map(replay => (
-      <React.Fragment key={replay.id}>
-        <Link
-          to={`/organizations/${organization.slug}/replays/${generateEventSlug({
-            project: replay.project,
-            id: replay.id,
-          })}/`}
-        >
-          <ReplayUserBadge
-            avatarSize={32}
-            displayName={replay['user.display']}
-            user={{
-              username: replay['user.display'],
-              id: replay['user.display'],
-              ip_address: replay['user.display'],
-              name: replay['user.display'],
-              email: replay['user.display'],
+    return replayList?.map(replay => {
+      const pathname = replay.url ? new URL(replay.url).pathname : '';
```

Comment:
Can we handle the case where `replay.url` is not a valid URL and `new URL()` throws? It might be better as a new function too. That would also make this diff a bit smaller since you can inline the function call when defining the `displayEmail` prop below and restore the implicit return.

Reply:
Looks like we have a similar function here https://github.com/getsentry/sentry/blob/master/static/app/components/events/errors.tsx#L106-L112 -- it might be worthwhile to make a follow-up PR that refactors the function into `app/utils`